instance/models/mixins/secret_keys.py

```diff
@@ -114,12 +141,45 @@ def secret_key(self):
         """
         return b64decode(self.secret_key_b64encoded)
 
+    @property
+    def rsa_key(self):
+        """
+        Return the RSA key as a Cryptodome.RSA object.
+        """
+        if not self.secret_key_rsa_private:
+            self.secret_key_rsa_private = generate_rsa_key(2048)
+            self.secret_key_rsa_private.save()
```

Comment:
@giovannicimolin, is there a `self.<field>.save()` method? How does it differ from `self.save()`. Also when calling this from an instance retrieved from the database, does it persist changes to the database and also update the instance from which it was called?